python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -401,6 +409,15 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
             "closest in time to the exposure.",
     )
 
+    useButlerCamera = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="If True, use a camera generated elsewhere in the pipeline to fit "
```

Comment:
Maybe be a bit more descriptive as to what you mean by "camera" (e.g. camera model/distortion model/object)